app-frontend/.eslintrc

```diff
@@ -0,0 +1,231 @@
+    "no-mixed-spaces-and-tabs": 1, // disallow mixed spaces and tabs for indentation
+    "no-multiple-empty-lines": [1, {"max": 2}], // disallow multiple empty lines (off by default)
+    "no-nested-ternary": 1, // disallow nested ternary expressions (off by default)
+    "no-new-object": 1, // disallow use of the Object constructor
+    "no-spaced-func": 1, // disallow space between function identifier and application
+    "no-ternary": 0, // disallow the use of ternary operators (off by default)
+    "no-trailing-spaces": 1, // disallow trailing whitespace at the end of lines
+    "no-underscore-dangle": 1, // disallow dangling underscores in identifiers
+    "one-var": [1, "never"], // allow just one var statement per function (off by default)
+    "operator-assignment": [1, "never"], // require assignment operator shorthand where possible or prohibit it entirely (off by default)
+    "padded-blocks": [1, "never"], // enforce padding within blocks (off by default)
+    "quote-props": [1, "as-needed"], // require quotes around object literal property names (off by default)
+    "quotes": [1, "single"], // specify whether double or single quotes should be used
+    "semi": [1, "always"], // require or disallow use of semicolons instead of ASI
+    "semi-spacing": [1, {"before": false, "after": true}], // enforce spacing before and after semicolons
+    "sort-vars": 0, // sort variables within the same declaration block (off by default)
+    "keyword-spacing": [1, {"before": true, "after": true}], // require a space after certain keywords (off by default)
+    "space-before-blocks": [1, "always"], // require or disallow space before blocks (off by default)
+    "space-before-function-paren": [1, {"anonymous": "always", "named": "never"}], // require or disallow space before function opening parenthesis (off by default)
+    "object-curly-spacing": [1, "never"], // require or disallow spaces inside brackets (off by default)
+    "array-bracket-spacing": [1, "never"], // require or disallow spaces inside brackets (off by default)
+    "computed-property-spacing": [1, "never"], // require or disallow spaces inside brackets (off by default)
+    "space-in-parens": [1, "never"], // require or disallow spaces inside parentheses (off by default)
+    "space-infix-ops": [1, {"int32Hint": false}], // require spaces around operators
+    "space-unary-ops": [1, {"words": true, "nonwords": false}], // Require or disallow spaces before/after unary operators (words on by default, nonwords off by default)
+    "spaced-comment": [1, "always"], // require or disallow a space immediately following the // in a line comment (off by default)
+    "wrap-regex": 0, // require regex literals to be wrapped in parentheses (off by default)
+
+    //
+    // ECMAScript 6
+    //
+    // These rules are only relevant to ES6 environments and are off by default.
+    //
+    "no-var": 2, // require let or const instead of var (off by default)
```

Comment:
If we're keeping this then the webpack configs need to be updated otherwise my linter is marking up those files a ton